workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -249,9 +249,10 @@ ideal_vgi_poi:
   endpoint: elements/count
   filter: (amenity=* or craft=* or emergency=* or historic=* or leisure=* or man_made=* or office=* or place=* or shop=* or tourism=*) and geometry:point
 
-ideal_vgi_lulc:
-  name: IDEAL-VGI Land Use and Land Cover
+lulc:
+  name: Land Use and Land Cover
   description: Features related to land use and land cover
   endpoint: elements/area
   filter: (landuse=* or natural=* or waterway=*) and geometry:polygon
+  source: https://www.vgiscience.org/projects/ideal-vgi.html
```

Comment:
@SlowMo24 could you have a brief look at the layer definition?
As you know we would like to add sources to the layer definitions. Is it fine for you to reference the IDEAL VGI project website here or is that not appropriate?

Reply:
ok, there are three approaches here: 

- The one you currently have is the easy one. But it includes errors and issues (see https://confluence.heigit.org/display/idealvgi/01+Data+extraction?preview=/90244886/58360018/TagsCountVsArea.pdf and https://heigit.org/a-basic-guide-to-osm-data-filtering/). In this case the website would be fine as source.
- The next easiest solution would be a list of 70 to 90 tags that I use. Here the website would also be fine, until we have a publication.
- The last option would be an adapted list from published research that also includes other keys like tourism, or amenity.

What do you prefer?

Reply:
Thanks for the feedback :)

Lets have the simple definition with a more verbose description for now. Something like this: 
```
    Features related to land use and land cover.
    Whereas this is a simple definition, it might lead to some issues.
    For further details please contact us.
```

Would you like to have a layer with a more advanced set of tags in OQT (Your suggest option 2 or 3)? I am happy to add one if you will use it.

Reply:
seems like I did not stress the **issues** part enough and the discrepancy of what we said in our blogpost to what we do ;-) . Please use option 2 with the following 58 tags:

`(landuse=allotments or landuse=animal_keeping or landuse=basin or landuse=brownfield or landuse=cemetery or landuse=churchyard or landuse=civic_admin or landuse=commercial or landuse=construction or landuse=depot or landuse=education or landuse=farmland or landuse=farmyard or landuse=forest or landuse=garages or landuse=grass or landuse=greenfield or landuse=greenhouse_horticulture or landuse=harbour or landuse=industrial or landuse=landfill or landuse=logging or landuse=meadow or landuse=military or landuse=orchard or landuse=plant_nursery or landuse=port or landuse=quarry or landuse=railway or landuse=recreation_ground or landuse=religious or landuse=reservoir or landuse=residential or landuse=retail or landuse=salt_pond or landuse=village_green or landuse=vineyard or natural=bare_rock or natural=beach or natural=dune or natural=fell or natural=glacier or natural=grassland or natural=heath or natural=landslide or natural=mud or natural=rock or natural=sand or natural=scree or natural=scrub or natural=shingle or natural=water or natural=wetland or natural=wood or waterway=boatyard or waterway=dam or waterway=dock or waterway=riverbank) and geometry:polygon`

and the description:

```
Features related to land use and land cover.
This definition includes all currently well established tags that define coherent areas at the most atomic level available. This definition therefore excluds areas with the keys aeroway, highway, leisure and tourism wich can be found in other sources.
For further details please contact us.
```

and the source

`https://osmlanduse.org, Fonte et al 2016 https://cartography-gis.com/docsbca/iccgis2016/ICCGIS2016-47.pdf, https://github.com/jasp382/glass/tree/master/core/glass/ete/osm2lulc, Schultz et al 2017 https://doi.org/10.1016/j.jag.2017.07.014, https://www.geog.uni-heidelberg.de/gis/ideal_en.html`

for the description and the source I am not sure about phrasing, format and content, so please adapt.